src/main/java/hudson/plugins/sshslaves/verifiers/MissingVerificationStrategyAdministrativeMonitor.java

```diff
@@ -55,4 +58,16 @@ public boolean isActivated() {
         return false;
     }
 
+    /**
+     * This method can be removed when the baseline is updated to 2.88
+     *
+     * @return If this version of the plugin is running on a Jenkins version where JENKINS-43786 is included.
+     */
+    @Restricted(DoNotUse.class)
+    public boolean isTheNewDesignAvailable() {
+        if (Jenkins.getVersion().isNewerThan(new VersionNumber("2.88"))) {
```

Comment:
This version needs to be updated to the actual one when upstream changes are integrated

Reply:
Sure, for that reason this PR is downstream of https://github.com/jenkinsci/jenkins/pull/2857